src/main/java/nl/tudelft/ti2206/group9/level/Track.java

```diff
@@ -193,6 +193,7 @@ public final synchronized void step() {
 		getPlayer().step();
 
 		distance += UNITS_PER_TICK;
+		State.addScore(UNITS_PER_TICK);
```

Comment:
Maybe we should still write a nice formula for this, but for now it's good enough :)